chia/full_node/block_store.py

```diff
@@ -57,7 +57,11 @@ async def create(cls, db_wrapper: DBWrapper):
 
         await self.db.execute("CREATE INDEX IF NOT EXISTS height on block_records(height)")
 
-        await self.db.execute("CREATE INDEX IF NOT EXISTS hh on block_records(header_hash)")
+        # earlier versions of chia created this index despite no lookups needing
+        # it. For now, just don't create it for new installs. In the future we
+        # may remove the index from existing installations as well
+        # await self.coin_record_db.execute("DROP INDEX IF EXISTS header_hash")
```

Comment:
I think you meant `DROP INDEX IF EXISTS hh` here, right?
I'll incorporate this in a PR I'm preparing